guide/13-managing-arcgis-applications/hub-basic-guide.ipynb

```diff
@@ -0,0 +1,673 @@
+       "                        </a>\n",
+       "                        <br/><img src='https://geosaurus.maps.arcgis.com/home/js/jsapi/esri/css/images/item_type_icons/layers16.png' style=\"vertical-align:middle;\">Hub Site Application by MMajumdar_geosaurus\n",
+       "                        <br/>Last Modified: December 12, 2022\n",
+       "                        <br/>0 comments, 0 views\n",
+       "                    </div>\n",
+       "                </div>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<Item title:\"Python Site\" type:Hub Site Application owner:MMajumdar_geosaurus>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "site1 = hub_basic.sites.add('Python Site')\n",
+    "site1.item"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d087c50",
+   "metadata": {},
+   "source": [
+    "You can attach web pages to your site to separate topics and to expand your site's navigation. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "4ca431d8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div class=\"item_container\" style=\"height: auto; overflow: hidden; border: 1px solid #cfcfcf; border-radius: 2px; background: #f6fafa; line-height: 1.21429em; padding: 10px;\">\n",
+       "                    <div class=\"item_left\" style=\"width: 210px; float: left;\">\n",
+       "                       <a href='https://geosaurus.maps.arcgis.com/home/item.html?id=5193511c814e4a989f67d44c9bdf0f8e' target='_blank'>\n",
+       "                        <img src='http://static.arcgis.com/images/desktopapp.png' class=\"itemThumbnail\">\n",
+       "                       </a>\n",
+       "                    </div>\n",
+       "\n",
+       "                    <div class=\"item_right\"     style=\"float: none; width: auto; overflow: hidden;\">\n",
+       "                        <a href='https://geosaurus.maps.arcgis.com/home/item.html?id=5193511c814e4a989f67d44c9bdf0f8e' target='_blank'><b>Python First Page</b>\n",
+       "                        </a>\n",
+       "                        <br/><img src='https://geosaurus.maps.arcgis.com/home/js/jsapi/esri/css/images/item_type_icons/layers16.png' style=\"vertical-align:middle;\">Hub Page by MMajumdar_geosaurus\n",
+       "                        <br/>Last Modified: December 12, 2022\n",
+       "                        <br/>0 comments, 0 views\n",
+       "                    </div>\n",
+       "                </div>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<Item title:\"Python First Page\" type:Hub Page owner:MMajumdar_geosaurus>"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "page1 = site1.pages.add('Python First Page')\n",
+    "page1.item"
+   ]
```

Comment:
* I built a local environment with the changes you've submitted in src repo #10145, and I receive this error when running the code to `add` a page (line 178-179):
```pytb---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
Cell In[9], line 1
----> 1 page1 = site1.pages.add('Python First Page')
      2 page1.item

File ~/Job/repos/geosaurus/src/arcgis/apps/hub/pages.py:331, in PageManager.add(self, title, site)
    329 page = Page(self._gis, item)
    330 # Link page to site
--> 331 status = site.pages.link(page)
    332 if status:
    333     return page

File ~/Job/repos/geosaurus/src/arcgis/apps/hub/pages.py:470, in PageManager.link(self, page, site, slug)
    468 # Update page and site data with new linking
    469 page.item.update(item_properties={"text": _page_data})
--> 470 return site.item.update(item_properties={"text": _site_data})

File ~/Job/repos/geosaurus/src/arcgis/gis/__init__.py:14526, in Item.update(self, item_properties, data, thumbnail, metadata)
  14523         fileName = self.name
  14524         item_properties["fileName"] = fileName
> 14526 ret = self._portal.update_item(
  14527     self.itemid,
  14528     item_properties,
  14529     data,
  14530     thumbnail,
  14531     metadata,
  14532     owner,
  14533     folder,
  14534     large_thumbnail,
  14535 )
  14536 if ret:
  14537     self._hydrate()

File ~/Job/repos/geosaurus/src/arcgis/gis/_impl/_portalpy.py:2649, in Portal.update_item(self, itemid, item_properties, data, thumbnail, metadata, owner, folder, large_thumbnail)
   2647 path += "/items/" + itemid + "/update"
   2648 if "text" in postdata:
-> 2649     resp = self.con.post_multipart(path, postdata, files)
   2650 else:
   2651     resp = self.con.post(path, postdata, files)

File ~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py:1266, in Connection.post_multipart(self, path, params, files, **kwargs)
   1264 if return_raw_response:
   1265     return resp
-> 1266 return self._handle_response(
   1267     resp=resp,
   1268     out_path=out_path,
   1269     file_name=file_name,
   1270     try_json=try_json,
   1271     force_bytes=kwargs.pop("force_bytes", False),
   1272 )

File ~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py:998, in Connection._handle_response(self, resp, file_name, out_path, try_json, force_bytes, ignore_error_key)
    996             return data
    997         errorcode = data["error"]["code"] if "code" in data["error"] else 0
--> 998         self._handle_json_error(data["error"], errorcode)
    999     return data
   1000 else:

File ~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py:1021, in Connection._handle_json_error(self, error, errorcode)
   1018                 # _log.error(errordetail)
   1020 errormessage = errormessage + "\n(Error Code: " + str(errorcode) + ")"
-> 1021 raise Exception(errormessage)

Exception: Item does not exist or is inaccessible.
(Error Code: 400)
```